gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -77,7 +77,13 @@ def add_arguments(self, parser):
               'large number of files are specified by input_pattern. '
               'Note that each VCF file must still contain valid header files '
               'even if this is provided.'))
-
+    parser.add_argument(
+        '--robust_header_extraction',
```

Comment:
I think something like `infer_undefined_headers` is a more representative name for this flag.

Reply:
Done